ja/messages.json

```diff
@@ -1559,13 +1559,13 @@
         "message": "Tampermonkey + スクリプトのバージョン"
     },
     "Tampermonkey_has_no_access_to_this_page": {
-        "message": "Tampermonkey のこのページへのアクセスが許可されていません"
+        "message": "Tampermonkey はこのページへのアクセス権なし"
     },
     "Tampermonkey_has_no_file_access_permission_": {
-        "message": "Tampermonkey のローカル ファイルへのアクセスが許可されていません!"
+        "message": "Tampermonkey にパソコン内ファイルへのアクセス権なし"
```

Comment:
「端末内ファイル」ではいかかでしょうか？ (How about "file in device"?)

Reply:
@shirayuki
You emphasis for Japanese, not transliteration. I have the same idea as you.

The transliteration "local" (ローカル) is a technical term for PC : [ローカル](https://kotobank.jp/word/%E3%83%AD%E3%83%BC%E3%82%AB%E3%83%AB-10013) (Koto bank) : The most commonly meaning in Japan is "countryside" (Antonym for Central city). And, It is unnatural or uncommon in other Japanese language. (e.g. 局所)

So I think of another word.
* Device (or Terminal) : The transliteration would be a more technical term ("デバイス"). Because public documents will say "端末". e.g. "携帯端末", "ATM端末"
* Personal Computer (PC): The Japanese omit this as "PersoCom" (パソコン). You probably think this means desktop or Windows. Because you say Android is not PC. That's a narrow meaning. However PC is an environment that has a GUI and runs apps. PC include Windows, Android, desktops, and smartphones. In a freely sense, it will be.

----

Because In 1977 when Windows didn't exist, Alan Kay published "Microelectronics and the Personal Computer" on *Scientific American*. It is likely that marketing was repeated to associate it with a particular product, Maybe. [Can a smartphone be a handheld personal computer?](https://www.quora.com/Can-a-smartphone-be-a-handheld-personal-computer) (Quora) Two experts see the smartphone as a personal computer.

Reply:
I agree with changing it to 'アクセス権なし'.
It's important to maintain consistency throughout the text, so it's preferable to translate all instances of 'local' as 'ローカル' to ensure uniformity.